packages/integrations/node/src/http-server.ts

```diff
@@ -23,21 +23,36 @@ function parsePathname(pathname: string, host: string | undefined, port: number)
 
 export function createServer(
 	{ client, port, host, removeBase }: CreateServerOptions,
-	handler: http.RequestListener
+	handler: http.RequestListener,
+	trailingSlash: string
 ) {
 	const listener: http.RequestListener = (req, res) => {
 		if (req.url) {
 			let pathname: string | undefined = removeBase(req.url);
 			pathname = pathname[0] === '/' ? pathname : '/' + pathname;
-			const encodedURI = parsePathname(pathname, host, port);
+			let encodedURI = parsePathname(pathname, host, port);
 
 			if (!encodedURI) {
 				res.writeHead(400);
 				res.end('Bad request.');
 				return res;
 			}
 
-			const stream = send(req, encodedURI, {
+			let pathForSend = encodedURI;
+
+			if (trailingSlash === 'never') {
+				if (pathname.endsWith('/')) {
+					encodedURI = parsePathname(pathname.substring(0, pathname.length - 1), host, port);
+					if (!encodedURI) {
+						res.writeHead(400);
+						res.end('Bad request.');
+						return res;
+					}
+				}
+				pathForSend = encodedURI + '/';
```

Comment:
Why are we adding a trailing slash if `trailingSlash` is `"never"`

Reply:
Hey @ematipico, we're adding a trailingSlash to pathForSend in case of trailingSlash is never so that the directory like behavior is maintained but URL redirection doesn't take place.